src/util.c

```diff
@@ -468,3 +469,15 @@ const char *pga_details(const PgAddr *a, char *dst, int dstlen)
 	}
 	return dst;
 }
+
+/*
+ * parser setter for parsing auth_dbname parameter using "cfparser"
+ */
+bool cf_set_authdb(struct CfValue *cv, const char *value)
```

Comment:
I do not have strong opinion on the correct placement of this parser. On one hand, most of the parsers are placed in the libusual. On the other hand, we can argue that this is not a primitive type we're parsing, but rather an information that is only relevant to PgBouncer. So, I guess that is fine.

+ I like the idea of having a validation during the startup + configuration reloads, though, would be nice to have a senior PgBouncer engineer's opinion on this. @JelteF , @eulerto  what do you think of this change?

Reply:
I think it makes sense to have it in pgbouncer itself, since indeed it's not a primitive type.